sdk/src/grpc-util.ts

```diff
@@ -62,8 +62,7 @@ export interface GrpcClientLookup {
  * @private
  */
 enum ServiceType {
-  SELF,
-  KALIX_SERVICE,
```

Comment:
This change is not good, the possible clients a user wants to look up are from the service itself (`SELF`) other deployed Kalix services in the same project (`KALIX_SERVICE`) and arbitrary gRCP services (`EXTERNAL`).

For the first two we need to send different set of auth headers. For external services we send no auth headers at all.

Reply:
Interesting. Thanks for the heads up. I expected something to break and alert me if this was really necessary. You saved me some precious time.

Reply:
I put it back.